.github/workflows/newman.yml

```diff
@@ -524,3 +523,91 @@ jobs:
           if [ -n "$summaries" ]; then
               echo -e "$summaries" >> $GITHUB_STEP_SUMMARY
           fi
+
+  Adding-testCases-keys:
+    if: ${{ contains(fromJson('["testing","unstable"]'), inputs.stability) }}
+    needs: [newman-test-list, associate-test-cases, newman-test-run]
+    runs-on: ubuntu-22.04
+    defaults:
+      run:
+        working-directory: centreon/tests/rest_api
+        shell: bash
+    steps:
+      - name: Checkout Repository
+        with:
+          ref: ${{ github.head_ref }}
```

Comment:
this will be broken once build will be triggered by `push` event because `head_ref` does not exist

Reply:
Could i change it by this : ref: ${{ github.event.before || github.sha }}